packages/conf/package.json

```diff
@@ -39,17 +39,5 @@
   "devDependencies": {
     "@lingui/jest-mocks": "*",
     "ts-node": "^10.9.1"
-  },
-  "peerDependencies": {
-    "ts-node": ">=10",
-    "typescript": ">=4"
-  },
-  "peerDependenciesMeta": {
-    "ts-node": {
-      "optional": true
-    },
-    "typescript": {
-      "optional": true
-    }
```

Comment:
Jiti claims they have zero dependencies. Don't know how that possible, because they use at least babel for transformation.

Reply:
Got that, they are bundle everything with webpack.